ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -64,13 +79,46 @@ const styles = StyleSheet.create({
 const LABEL_COL_SIZE_NARROW_LABEL = 1;
 const VALUE_COL_SIZE_NARROW_LABEL = 2;
 
-const LABEL_COL_SIZE_WIDE_LABEL = 1;
+const LABEL_COL_SIZE_WIDE_LABEL = 2;
 const VALUE_COL_SIZE_WIDE_LABEL = 1;
 
 /**
  * Details of transaction
  */
 export class TransactionDetailsScreen extends React.Component<Props, never> {
+  /**
+   * It sum the amount to pay and the fee requested to perform the transaction
+   * TO DO: it could be provided by API as presentd on header
+   */
+  private getTotalAmount(transaction: Readonly<WalletTransaction>) {
+    return +(transaction.amount + transaction.transactionCost).toFixed(12);
+  }
+
+  private getsubHeader() {
+    return this.props.transaction.isTransactionCompleted ? (
+      <View>
+        <Grid>
+          <Col size={1} />
+          <Col size={5} style={WalletStyles.alignCenter}>
+            <View spacer={true} />
+            <Row>
+              <H1 style={WalletStyles.white}>{I18n.t("wallet.thanks")}</H1>
```

Comment:
We are missing the tick mark here (✔). We should integrate it as a unicode glyph. [Here](https://en.wikipedia.org/wiki/Check_mark) you'll find some Unicode options. U+2714 looks good to me

Reply:
sounds good!